prod/mach_zehnder_gadget/javascript/app/WebSocketController.js

```diff
@@ -161,8 +240,24 @@ myApp.controller('WebSocketController',['$scope', function($scope) {
         }
         if($("#laserButton").hasClass("btn-warning")){
             ws.send("laser_power?"+1);
+            graphFlag=1;
+
+            //Log Activity
+             var logObject = {
+                "objectType": "button",
+                "displayName":"laserButton",
+             }
+             actionLogger.logStart(logObject);
+
         }else{
```

Comment:
add space before and after else. for the coding conventions check: https://github.com/react-epfl/graaspeu/wiki/Coding-And-Naming-Conventions